src/sentry/sentry_metrics/configuration.py

```diff
@@ -131,3 +136,51 @@ def get_ingest_config(
         )
 
     return _METRICS_INGEST_CONFIG_BY_USE_CASE[(use_case_key, db_backend)]
+
+
+def initialize_sentry_and_global_consumer_state(config: MetricsIngestConfiguration) -> None:
+    """
+    Initialization function for subprocesses spawned by the parallel indexer.
+
+    It does the same thing as `initialize_global_consumer_state` except it
+    initializes the Sentry Django app from scratch as well.
+
+    `config` is pickleable, and this function lives in a module that can be
+    imported without any upfront initialization of the Django app. Meaning that
+    an object like
+    `functools.partial(initialize_sentry_and_global_consumer_state, config)` is
+    pickleable as well (which we pass as initialization callback to arroyo).
+    """
+    from sentry.runner import configure
+
+    configure()
+
+    initialize_global_consumer_state(config)
+
+
+def initialize_global_consumer_state(config: MetricsIngestConfiguration) -> None:
+    """
+    Initialization function for the main process of the metrics indexer.
+
+    This primarily sets global tags for instrumentation in both our
+    statsd/metrics usage and the Sentry SDK.
+    """
+
+    import sentry_sdk
+    from arroyo import configure_metrics
```

Comment:
What is the reason for not importing this at the top of the module ? It should not be a problem for the multi process consumer.

Reply:
ok, i moved those two imports to the t